### PR TITLE
An003 string array list

### PR DESCRIPTION
**Pull request by anusha-grao28**
the programs on arraylist, set, arrays, map have been done

### CODE REVIEW

File: src/main/java/com/anusha/collectiontasks/AN005_SetProg.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+       number.add(40);
+       number.add(20);
+
+       System.out.println(" my set of integers are:\n");
+       System.out.println(number);
+       System.out.println(" size of an array is: "+number.size());
+
+       //2. Test how many 20's are present in the collection
+        boolean result=number.contains(20);
+        System.out.println(" to count value 20 in the set is: "+result);
```
Comment (gautirao) [inline src/main/java/com/anusha/collectiontasks/AN005_SetProg.java:25]: please change this message

File: src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        System.out.println("Method-II for displaying map contents: \n"+ Collections.singletonList(hmap));
+        System.out.println("----------------\n");
+        //3. Method-III
+        for (Map.Entry<Integer, String> entry : hmap.entrySet()) {
+            System.out.println(entry.getKey()+" : "+entry.getValue());
+        }
+        System.out.println("----------------\n");
+        //3. to print the third element of the map
+        System.out.println(" the elements of the map is: "+hmap.size());
+        System.out.println("the third element from the map is: "+hmap.get(3));
```
Comment (gautirao) [inline src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java]: whats the ouput t does.get (3) give you the fourth  or third element. can you verify 
```suggestion
        System.out.println("the third element from the map is: "+hmap.get(2));
```

**Review by gautirao — CHANGES_REQUESTED** [review]
can you have a look at the review comments

File: src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java (hunk 2 above)
Comment (anusha-grao28) [reply src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java]: get(3) give output of third element from the map

Comment (gautirao) [reply src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java]: can you test and print output here .

Comment (gautirao) [reply src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java]: can you change it to use iterators. May be do it in another commit

Comment (anusha-grao28) [reply src/main/java/com/anusha/collectiontasks/AN006_MapsProgs.java]: ok i will change using iterator

**Pull request merged**